lib/fake_braintree/sinatra_app.rb

```diff
@@ -6,6 +6,7 @@
 require 'fake_braintree/credit_card'
 require 'fake_braintree/address'
 require 'fake_braintree/payment_method'
+require 'fake_braintree/transaction'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.